src/components/date-range/date-range.d.ts

```diff
@@ -6,20 +6,31 @@ export interface DateRangeChangeEvent {
   target: {
     value: [
       {
-        formattedValue: number;
+        formattedValue: string;
         rawValue: string;
       },
       {
-        formattedValue: number;
+        formattedValue: string;
         rawValue: string;
       }
     ];
   };
 }
 
+export interface DateInputRefMap {
```

Comment:
If we want to export these all from Carbon we're better off having a `export * from "./date-range";` in the index.d.ts file as well as the default export, if not we can just remove the exports here